src/components/VContentSwitcher/VContentSwitcherPopover.vue

```diff
@@ -9,9 +9,11 @@
       <VContentSwitcherButton
         :a11y-props="a11yProps"
         :active-item="activeItem"
+        aria-controls="content-switcher-popover"
       />
     </template>
     <VContentTypes
+      id="content-switcher-popover"
```

Comment:
a sign of another  a11y problem  a good idea. To be honest, I've read about `aria-controls` and aria-haspopup, and still don't understand whether it should be dialog or menu, and whether aria-controls is strictly necessary, @sarayourfriend .
I added it here because I couldn't find a better more accessible way of selecting the content switcher in e2e tests (the text changes, so I can't use it). And this might bea sign of another  a11y problem

Reply:
I also was confused about that but looking at Reakit, as I often do to check on ideas for how to implement some of these patterns correctly, it does include `aria-controls` in the `Disclosure` component which is used for Dialog as well in `DialogDisclosure`. So I think you're right to add it.